sql/snapshot.sql

```diff
@@ -9,9 +30,8 @@ CREATE TABLE block (
 	hash varchar NOT NULL,
 	height int8 NOT NULL,
 	parent jsonb NULL,
-	lock_timestamp timestamp NOT NULL,
-	snapshot_hash varchar NOT NULL,
-	snapshot_ordinal int8 NOT NULL,
+	created_at timestamp DEFAULT now() NOT NULL,
+	updated_at timestamp DEFAULT now() NOT NULL,
```

Comment:
I'll just comment here but it applies to all the tables. You'll need an auto-update trigger to support `updated_at`.